src/pages/Metrics/MetricsPage.scss

```diff
@@ -0,0 +1,344 @@
+.metrics-page {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding-top: $top-menu-height;
+  min-height: 100vh;
+
+  &__wrapper {
+    @include desktop {
+      display: flex;
+      justify-content: space-between;
+      width: 100%;
+    }
+  }
+
+  &__container {
+    display: block;
+
+    @include desktop {
+      display: flex;
+      justify-content: space-between;
+    }
+  }
+
+  &__image-container {
+    @include desktop {
+      margin-right: 48px;
+      width: 100%;
+      max-width: 448px;
+    }
+  }
+
+  &__image {
+    @include noselect;
+
+    width: 100%;
+
+    @include tablet {
+      width: none;
+    }
+  }
+
+  &__metric {
+    display: flex;
+    flex-direction: column;
+    margin-top: 24px;
+    width: 100%;
+
+    @include desktop {
+      margin-top: 0;
+    }
+  }
+
+  &__title {
+    margin-bottom: 16px;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 1.4;
+    text-align: left;
+  }
+
+  &__uniqueness {
+    margin-bottom: 28px;
+  }
+
+  &__uniqueness-bars {
+    display: flex;
+    align-items: center;
+    border-radius: 20px;
+    padding: 16px;
+    box-shadow: 1px 4px 16px rgb(180 173 189 / 15%);
+
+    @include desktop {
+      padding: 16px 24px;
+    }
+  }
+
+  &__stroke-bars {
+    display: flex;
+    flex-direction: column;
+    margin-left: 24px;
+    width: 100%;
+
+    @include desktop {
+      margin-left: 48px;
+    }
+  }
+
+  &__box {
+    display: flex;
+    align-items: flex-start;
+    flex-direction: row;
+    margin-bottom: 16px;
+    width: 100%;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__features-item {
+    margin-right: 8px;
+    margin-bottom: 8px;
+    border-radius: 8px;
+    padding: 8px 16px;
+    font-weight: 500;
+    font-size: 14px;
```

Comment:
Please check your files for redundant typography rules (e.g. font-weight, font-size, etc.), which are already set globally